src/HealthGPS.Datastore/datamanager.cpp

```diff
@@ -21,18 +21,9 @@ nlohmann::json read_input_files_from_directory(const std::filesystem::path &root
             fmt::format("File-based store, index file: '{}' not found.", full_filename.string()));
     }
 
+    // Read in JSON file
     auto index = nlohmann::json::parse(ifs);
 
-    if (!index.contains("version")) {
-        throw std::runtime_error("File-based store, invalid definition missing schema version");
-    }
-
-    auto version = index["version"].get<int>();
-    if (version != 2) {
-        throw std::runtime_error(fmt::format(
-            "File-based store, index schema version: {} mismatch, supported: 2", version));
-    }
-
     // Validate against schema
     ifs.seekg(0);
     const auto schema_directory = hgps::get_program_directory() / "schemas" / "v1";
```

Comment:
If the information on the schema to use is included in the `index.json` file, shall you use that instead?

Reply:
I don't actually want users to be able to use different schemas. My current plan is to enforce that they must use the latest schema with the latest version of Health-GPS, but at some point we might want to revisit this for the sake of backwards compatibility.